blog/top-5-cybersecurity-incidents-in-cicd/index.md

```diff
@@ -0,0 +1,78 @@
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-1: Insufficient Flow Control Mechanisms](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-01-Insufficient-Flow-Control-Mechanisms)  
+- [CICD-SEC-10: Insufficient Logging and Visibility](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-10-Insufficient-Logging-And-Visibility)
+
+**References:**  
+- [SolarWinds hack timeline: Who knew what, and when (CSO Online)](https://www.csoonline.com/article/3620961/solarwinds-hack-timeline-who-knew-what-and-when.html)  
+
+---
+
+**2. Codecov Bash Uploader Incident (2021)**  
+Codecov, a code coverage tool, was compromised after attackers modified the Bash Uploader script used in its CI/CD process. The alteration allowed the exfiltration of sensitive data like credentials and environment variables from thousands of customers.
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-6: Insufficient Credential Hygiene](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-06-Insufficient-Credential-Hygiene)  
+- [CICD-SEC-9: Improper Artifact Integrity Validation](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-09-Improper-Artifact-Integrity-Validation)
+
+**References:**  
+- [Codecov Bash Uploader Script Breach: What You Need to Know (Security Week)](https://www.securityweek.com/codecov-bash-uploader-script-breach-what-you-need-know)  
+
+---
+
+**3. Travis CI Secrets Leak (2021)**  
+Travis CI, a widely used CI service, was found to have exposed secrets, including tokens and keys, from thousands of repositories due to a vulnerability in the service’s API. This exposure affected many open-source projects.
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-4: Poisoned Pipeline Execution](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-04-Poisoned-Pipeline-Execution)  
+- [CICD-SEC-6: Insufficient Credential Hygiene](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-06-Insufficient-Credential-Hygiene)
+
+**References:**  
+- [Travis CI security vulnerability impacts open source software (The Hacker News)](https://thehackernews.com/2021/09/travis-ci-flaw-exposes-secrets-of.html)  
+
+---
+
+**4. PHP Git Server Compromise (2021)**  
+Attackers compromised the PHP Git repository and attempted to add a backdoor to the PHP source code by committing changes through the compromised server. This incident highlighted vulnerabilities in the version control and CI/CD processes.
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-7: Insecure System Configuration](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-07-Insecure-System-Configuration)  
+- [CICD-SEC-9: Improper Artifact Integrity Validation](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-09-Improper-Artifact-Integrity-Validation)
```

Comment:
Need details on 
- Need more details on "how" they compromised the repo (what supply chain weakness was involved ?)
to verify theses risks

Reply:
Changed in the latest commit; should be validated in a new review.

Reply:
Ok for me 👌